packages/server/__snapshots__/4_request_spec.ts.js

```diff
@@ -243,7 +243,8 @@ Headers: {
   "content-length": "19",
   "etag": "W/13-52060a5f",
   "date": "Fri, 18 Aug 2017    XX:XX GMT",
-  "connection": "keep-alive"
+  "connection": "keep-alive",
+  "keep-alive": "timeout=5"
```

Comment:
Node.js adds this header by default to keep-alive connections now to fix a potential race condition. This is ok because as a proxy, we are entitled to add hop-by-hop headers like Connection and Keep-Alive.